dpctl/tests/elementwise/test_exp.py

```diff
@@ -112,20 +131,48 @@ def test_exp_values(dtype):
     assert_allclose(dpt.asnumpy(dpt.exp(X)), np.exp(Xnp), atol=tol, rtol=tol)
 
 
-@pytest.mark.parametrize("dtype", ["e", "f", "d"])
-def test_exp_special_cases(dtype):
+@pytest.mark.parametrize("dtype", ["f2", "f4", "f8"])
+def test_exp_real_special_cases(dtype):
     q = get_queue_or_skip()
     skip_if_dtype_not_supported(dtype, q)
 
     tol = 8 * dpt.finfo(dtype).resolution
     x = [np.nan, -np.nan, np.inf, -np.inf, -1.0, 1.0, 0.0, -0.0]
     Xnp = np.array(x, dtype=dtype)
     X = dpt.asarray(x, dtype=dtype)
-    assert_allclose(dpt.asnumpy(dpt.exp(X)), np.exp(Xnp), atol=tol, rtol=tol)
+
+    Y = dpt.asnumpy(dpt.exp(X))
+    Ynp = np.exp(Xnp)
+    assert_allclose(Y, Ynp, atol=tol, rtol=tol)
+    assert_array_equal(np.signbit(Y), np.signbit(Ynp))
+
+
+@pytest.mark.parametrize("dtype", ["f2", "f4", "f8"])
+def test_exp_real_strided(dtype):
+    q = get_queue_or_skip()
+    skip_if_dtype_not_supported(dtype, q)
+
+    np.random.seed(42)
+    strides = np.array([-4, -3, -2, -1, 1, 2, 3, 4])
+    sizes = np.arange(2, 100)
```

Comment:
I we think we should judiciously choose small number of `sizes`. I propose `[2, 4, 6, 8, 9, 24, 72]`. This reduces the test suite runtime. 

It is a good idea to run tests with `--durrations=N`, with `N` being a small number to identify slowest tests and try to improve their performance.